lib/elixir/lib/kernel/cli.ex

```diff
@@ -109,6 +109,15 @@ defmodule Kernel.CLI do
     kind, reason -> {kind, reason, __STACKTRACE__}
   end
 
+  @doc """
+  Function invoked across nodes for `--rpc-call`.
+  """
+  def rpc_call(mod, fun, args) do
+    wrapper(fn -> apply(mod, fun, [args]) end)
```

Comment:
You can rewrite the processing of rpc_eval as part of rpc_call. rpc_eval is the same as calling `rpc_call(Code, :eval_string, expr)`.